mettle/worker/redis.go

```diff
@@ -95,7 +95,7 @@ func (r *redisClient) ReadBlob(dg *pb.Digest) ([]byte, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), r.timeout)
 	defer cancel()
 
-	if r.limiter.Allow() {
+	if r.limiter.Tokens() < 1.0 {
```

Comment:
>= 1.0, no?

(the other looks correct)

Reply:
Ah, yes. Mistakenly did both at the same time.